src/gui/preferences.c

```diff
@@ -379,6 +393,33 @@ static void init_tab_general(GtkWidget *stack)
   gtk_spin_button_set_value(GTK_SPIN_BUTTON(fontsize), dt_conf_get_float("font_size"));
   g_signal_connect(G_OBJECT(fontsize), "value_changed", G_CALLBACK(font_size_changed_callback), 0);
 
+  GtkWidget *screen_ppd_overwrite = gtk_spin_button_new_with_range(-1.0f, 8.0f, 0.2f);
+  label = gtk_label_new(_("images DPI scaling factor"));
```

Comment:
images -> thumbs

Which will be less ambiguous I think.

Reply:
done, I have also reworked the other label to make the difference more obvious